meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -53,15 +53,15 @@ export interface MongoSegment {
 }
 
 export interface MongoPart {
-	_id: string
+	_id: any
```

Comment:
I would very much like to have the type be `ObjectId`. However as our tests are at the moment, it will force all tests that wants to set an id, to later perform actions on, to follow some strict rules about the string. The tests must then supply a string to the id that is either 12 bytes, or 24 hex characters, otherwise the following error gets thrown.

`BSONTypeError: Argument passed in must be a string of 12 bytes or a string of 24 hex characters or an integer`

Reply:
Would like to hear @LindvedKrvang input on this before making the changes needed. 
That said, i am fine with making the change, even though it is going to cause alot of compile time type errors that i'd need to fix.

Reply:
Are you adding `@types/mongodb` to `dependencies` or `devDependencies` and which package.json are you adding it to?

Reply:
Now that you mention it, it might have been added to the wrong section.
It was added to the package.json under meteor.

Reply:
Not sure why i originally added it, as the ObjectId appear to also be inside the `mongodb` package.

I can confirm with high propability that the package was added to the wrong section. Woops.
Regardless the `@types/mongodb` has also been deprecated, and is not part of the project.

Reply:
I still think we should `ObjectId` from the mongodb package.

Reply:
Type for `_id` has been updated to ObjectId. 
Test code effected by this change, has also been adjusted to conform to the requriments for creating an ObjectId.